node/resolvers/ReturnRequestResponse.ts

```diff
@@ -131,19 +131,28 @@ export const ReturnRequestResponse = {
     return refundData
   },
   items: async (root: ReturnRequest, _args: unknown, ctx: Context) => {
-    const { id, items } = root
-
-    if (items) return items
+    const { id } = root
 
     const {
-      clients: { returnRequest: returnRequestClient },
+      clients: { returnRequest: returnRequestClient, oms },
     } = ctx
 
-    const { items: itemsList } = await returnRequestClient.get(id as string, [
-      'items',
-    ])
+    const { orderId, items: itemsList } = await returnRequestClient.get(
+      id as string,
+      ['orderId', 'items']
+    )
```

Comment:
We don't need to make another request to get the request. It is passed into this function as `root`, hence the type it has. Moreover, not sure the order should be the source of truth here. @cesarocampov told me about a endpoint that brings the seller name giving the seller id.